### ISSUE TITLE
Is it possible to use django-modeltranslation outside of admin?

### ISSUE THREAD

**Issue by NogerbekNurzhan** [deschler/django-modeltranslation#425]
Hello! In my Django project I want to use your app. I successfully install app and make settings, also registered models for translation as it was recommended in docs.

**Question**: Is it possible to use this app outside of admin? If it possible what I need to do? I am little bit comfused.

**translation.py:**
```
class ArticleTranslationOptions(TranslationOptions):
    fields = ('title', 'body',)

translator.register(Article, ArticleTranslationOptions)
```

**Comment by madEng84** [comment]
Yes, you can.
In read mode you will see the translated field
In write mode, if you want to set a field for every language, you have to customize your form __init__ to add the field for every language that you have enabled in your settings.
Same for serializers if you are using djangorestframework

The registration you have setted in translation.py enables the migrations to chenge your db, not the admin visualization, that is enabled by inheriting you ModelAdmin from ModelTranslationAdmin

**Comment by NogerbekNurzhan** [comment]
I am interested in write mode (Create/Edit Form). Is this code would be enough or I need to do something else?

**forms.py:**
```
from news.models import Article
from modeltranslation.forms import TranslationModelForm

class ArticleForm(TranslationModelForm):
    class Meta:
        model = Article
```

**Comment by NogerbekNurzhan** [comment]
When I try to use _ModelForms_ instead of _TranslationModelForm_ it shows in form only fields: title and body. How to show other fields in form?

**Comment by zlorf** [comment]
If you want to show all fields, use `ModelForm`. `TranslationModelForm` is intended to hide all transaltion fields:
http://django-modeltranslation.readthedocs.io/en/latest/forms.html
Also please check if form is not being registered too early: http://django-modeltranslation.readthedocs.io/en/latest/registration.html#precautions-regarding-registration-approach

**Comment by NogerbekNurzhan** [comment]
Right now I have fields in my forms but steal have strange problem in edit form.

_title_ is original field (Russian). Other fields are title_en (English), title_de (German). When I open edit form _title_ and _title_en_ fields are the same despite the title field must be in Russian but in fact they both are in English. Why _title_ field duplicate the value of _title_en_? In the same time _title_de_ is correct. How to change this strange behavior?

**Comment by zlorf** [comment]
Use `title_ru`. The original field (`title`) is not containing any specific language. It rather shows/writes current active language.

**Comment by NogerbekNurzhan** [comment]
Well, I can use _title_ru_ field. Its not problem, but can i stop overwrite of _title_ field?!

**Comment by zlorf** [comment]
No, you cannot stop it. Again, it's in the docs: http://django-modeltranslation.readthedocs.io/en/latest/usage.html#the-state-of-the-original-field
